image/docker_schema2_test.go

```diff
@@ -374,6 +374,30 @@ func (d *memoryImageDest) PutBlob(stream io.Reader, inputInfo types.BlobInfo) (t
 	d.storedBlobs[inputInfo.Digest] = contents
 	return types.BlobInfo{Digest: inputInfo.Digest, Size: int64(len(contents))}, nil
 }
+func (d *memoryImageDest) HasBlob(inputInfo types.BlobInfo) (bool, int64, error) {
+	if inputInfo.Digest.String() == "" {
+		panic("inputInfo.Digest unexpectedly empty")
+	}
+	if d.storedBlobs == nil {
+		d.storedBlobs = make(map[digest.Digest][]byte)
```

Comment:
we can be pretty confident that `d.storedBlobs` isn't containing anything at this point and so, we could return directly below this line, right?